be/src/vec/exec/format/parquet/parquet_pred_cmp.h

```diff
@@ -372,7 +379,8 @@ class ParquetPredicate {
     static bool filter_by_stats(const ColumnValueRangeType& col_val_range,
                                 const FieldSchema* col_schema, bool is_set_min_max,
                                 const std::string& encoded_min, const std::string& encoded_max,
-                                bool is_all_null, const cctz::time_zone& ctz) {
+                                bool is_all_null, const cctz::time_zone& ctz,
+                                bool use_min_max_value = false) {
```

Comment:
How about take `tparquet::Statistics` as parameters? In this way, the fix process can be encapsulated inside the function instead of being judged externally, and there is no need to pass so many parameters.

Reply:
ok, I will refractor it in the next PR.